enterprise/app/filter/filter.tsx

```diff
@@ -207,6 +267,11 @@ export default class FilterComponent extends React.Component<FilterProps, State>
             {selectedRoles.has("") && <span className="role-badge DEFAULT">Default</span>}
             {selectedRoles.has("CI") && <span className="role-badge CI">CI</span>}
             {selectedRoles.has("CI_RUNNER") && <span className="role-badge CI_RUNNER">Workflow</span>}
+            {userValue && <span className="advanced-badge">{userValue}</span>}
```

Comment:
ubernit: maybe add a todo here to add icons or a prefix label?

Reply:
Just added the icons instead of the todo